scripts/repo-alignment-doctor.ps1

```
@@ -0,0 +1,234 @@
+# Checkout context
+if ($ctx.IsWorktree) {
+  Add-Check 'checkout_context' 'ok' "Worktree checkout ($($ctx.RepoRoot))" ''
+}
+else {
+  Add-Check 'checkout_context' 'warn' "Main checkout - feature work belongs under $($ctx.DevRootPath)" '.\scripts\migrate-main-to-worktree.ps1 -Name my-task -Owner cursor'
+}
+
+# Canonical remote
+$originUrl = (git -C $ctx.RepoRoot remote get-url origin 2>$null).Trim()
+if ($originUrl -match 'modme-ui-01') {
+  Add-Check 'remote_canonical' 'ok' "origin matches GitHub canonical ($originUrl)" ''
```

PSScriptAnalyzer (1.25.0)

[info] Cmdlet 'Add-Check' has positional parameter. Please use named parameters instead of positional parameters when calling a command.

(PSAvoidUsingPositionalParameters)